### DISCUSSION TITLE
Future of this project

### DISCUSSION THREAD

**Discussion by bluskript** [windicss/windicss#155] [Q&A] [answered]
[Recently, TailwindCSS has announced its own just-in-time CSS compiler](https://github.com/tailwindlabs/tailwindcss-jit), which reduces bundle size for dev significantly. This pretty much the same technology as WindiCSS uses, so now i'm curious what the future of this project is (e.g. will this still be maintained? Will windi have any new features that tailwind doesn't have yet?).

**Comment by edimitchel** [comment, marked as answer]
You will find the answer here : https://github.com/windicss/windicss/discussions/176#discussioncomment-506195
